src/Servers/IIS/AspNetCoreModuleV2/CommonLib/ServerErrorHandler.h

```diff
@@ -9,31 +9,40 @@
 class ServerErrorHandler : public REQUEST_HANDLER
 {
 public:
+    ServerErrorHandler(IHttpContext& pContext, USHORT statusCode, USHORT subStatusCode, std::string statusText, HRESULT hr, HINSTANCE module, bool disableStartupPage, BYTE* content, int length) noexcept
```

Comment:
nit: use `const std::string&` instead of `std::string` to avoid the string copies.